sources/osg/WebGLCaps.js

```diff
@@ -363,6 +383,7 @@ WebGLCaps.prototype = {
             }
 
             ext[ sup ] = gl.getExtension( sup );
+            this.getAndApplyExtension( gl, sup );
```

Comment:
Does it really works for all extension ? If signature between extension and core of webgl2 it will not work